tests/test_geniza.py

```diff
@@ -55,6 +55,44 @@ def test_extract_stock_v1(self):
             )
             self.assertEqual(['MSFT'], symbols)
 
+    @urlmatch(path=r'/v1/detectors/language$')
```

Comment:
Can we add a test in another language?

Reply:
I am not apposed to a test in another language... but I don't think it is strictly necessary here.  All the language handling happens server-side, this is just testing the interaction with the server.